src/containers/pages/IRS/assignments/index.tsx

```diff
@@ -153,7 +153,8 @@ const mapStateToProps = (state: Partial<Store>): DispatchedStateProps => {
   const planStatus = [PlanStatus.ACTIVE];
   const fiPlans = getPlanRecordsArray(state, InterventionType.FI, planStatus);
   const irsPlans = getPlanRecordsArray(state, InterventionType.IRS, planStatus);
-  const plans = [...fiPlans, ...irsPlans].sort(
+  const mdaPlans = getPlanRecordsArray(state, InterventionType.MDA, planStatus);
+  const plans = [...fiPlans, ...irsPlans, ...mdaPlans].sort(
     (a: PlanRecord, b: PlanRecord) => Date.parse(b.plan_date) - Date.parse(a.plan_date)
   );
```

Comment:
Would be nice if we could just allow all types of plans render, though I'm not sure how to iterate over an `enum` like this. I think the argument against passing `null` instead of the plan type was to make sure and only ever show legitimate plans.

Reply:
> Would be nice if we could just allow all types of plans render,

Yeap

>  though I'm not sure how to iterate over an `enum` like this.

Not sure if the you are talking about the code itself, in which case i think you could do 
```Object.values``` or a `for ... in` loop.

> I think the argument against passing `null` instead of the plan type was to make sure and only ever show legitimate plans

yeah makes sense, maybe we can put the enum iteration in the `getPlanRecordsArray` and then have all the different InterventionTypes as the default argument. Adding more intervention types would require we only add to the said argument. I worked on something like this a while back #553 and i went with using a default of null did not know then about __make sure and only ever show legitimate plans.__

Reply:
>  maybe we can put the enum iteration in the

yeah, sorry, no iteration would be required in this case.